less/home-page.less

```diff
@@ -45,31 +85,118 @@
     padding: 50px 60px;
     display: flex;
     justify-content: center;
-
+    @media @mobile {
+      
+      width: 100%;
+      padding-top: 20px;
+      padding-bottom: 20px;
+    
+    }
+  
     .walk-text {
       width: 50%;
+      @media @mobile {
+        width: 100%;
+        
+      }
 
       h3 {
         margin-bottom: 10px;
+        @media @mobile {
+         width: 100%;
+         margin-top: 40px;
+         font-size: 2.2rem;
+        }
+        
       }
 
       p {
         margin-bottom: 26px;
+        
       }
 
       .btn {
+        .custom-btn(100px, 50px, @btn-bg-color, @sea-shell);
+      
+        @media @mobile {
+          .custom-btn(300px, 50px, @btn-bg-color, @sea-shell);
+          
+        }
+  
+      }
+      @media @mobile{
+        order: 0;
+      }
+      @media @mobile{
+        order: 1;
+        
       }
     }
 
     .walk-img {
       width: 46%;
       margin-left: 4%;
-      
+      @media @mobile {
+        width: 100%;
+      }
       img {
         width: 100%;
         height: auto;
         border-radius: 10px;
+        @media @mobile { 
+          order: 1;
+          width: 100%;
+        }
       }
     }
+    @media @mobile {
+      flex-direction: column;
+      width: 100%;
+
+    }
+  }
+  .register {
+    padding: 20px;
+    background-color: @wild-sand;
+  input[type=text], input[type=password] {
+    width: 100%;
+    padding: 15px;
+    margin: 5px 0 22px 0;
+    font-size: 1.2rem;
+    border: none;
+    background: @waterloo;
```

Comment:
This looks great, I'm glad that you continued working on this project and got some stretch done! If you would like to get 3 stars on this project, please submit another sprint retrospective, let me know, and I'll update your score

Reply:
Thank you! I have just submitted a new sprint retrospective.